bindings/python/test/trajectory/orbit/models/test_meoe.py

```
@@ -0,0 +1,411 @@
+            Length.kilometers(7501.0), meoe_default_values["f"], meoe_default_values["g"],
+            meoe_default_values["h"], meoe_default_values["k"], meoe_default_values["L"]
+        )
+
+        assert meoe_object == meoe_copy
+        assert meoe_object != meoe_different_p
+        assert (meoe_object == MEOE.undefined()) is False
+        assert (MEOE.undefined() == MEOE.undefined()) is False # Undefined is not equal to undefined
+
+
+    def test_string_representations(self, meoe_object: MEOE, meoe_default_values):
+        assert str(meoe_object) == f"MEOE (p: {meoe_default_values['p'].to_string()}, f: {meoe_default_values['f']:.6f}, g: {meoe_default_values['g']:.6f}, h: {meoe_default_values['h']:.6f}, k: {meoe_default_values['k']:.6f}, L: {meoe_default_values['L'].to_string(Angle.Unit.Degree)} [deg])" # Comparing with decorator
```

Pylint (3.3.7)

[convention] Line too long (319/100)

(C0301)